### DISCUSSION TITLE
Furnace

### DISCUSSION THREAD

**Discussion by opekope2** [opekope2/StaffMod#14] [Staff blocks]
Furnace is shown at [00:41-00:45](https://youtu.be/vlVUVsx6BYY?t=41) smelting items when hovered over them. It also gives the player XP.

Implementation idea: press and hold the use key (right click) to ignite the furnace. While the furnace is working, it smelts items in front of the player
Attack damage: same as empty staff (?)
Equip time: same as empty staff (?)

**Comment by charmander9695** [comment]
how about in a three block radius

**Reply by BoB10011** [reply]
oh yeah and in the EP the food and ores got smelted instantly

**Reply by opekope2** [reply]
I think those were stacks of 1. Smelting takes time in the mod to balance it for survival gameplay (staff crafting coming this spring). It's still 200x faster than a furnace, and 100x faster than a smoker or blast furnace as of now.

**Comment by opekope2** [comment]
Smelting everything is a bit too OP, and may unintentionally smelt things the player didn't want to.
The item in the staff model is about 1.75 blocks away from the player (third person). I was thinking about smelting the closest item to the furnace, if it's close enough.

**Reply by charmander9695** [reply]
you are right about the  smelting everything but there are three types of furnace, so can split groups of items with each furnace

p.s i just give out ideas in case you ran dry

**Reply by opekope2** [reply]
I completely forgot about the other two, great idea. However, I'd allow the furnace to smelt everything (easier this way, I only have to check one registry), maybe throttle its speed a bit.

**Comment by opekope2** [comment]
Further in a future release: require the furnace to have fuel in order to function. This would be a nerf for more realistic survival experience
